### PR TITLE
Update UI

### PR DESCRIPTION
**Pull request by rionshocker**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/contact/FindContactCommand.java [hunk 1]
```diff
@@ -30,9 +33,23 @@ public FindContactCommand(NameContainsKeywordsPredicate predicate) {
...
     public CommandResult execute(Model model) {
         requireNonNull(model);
-        model.updateFilteredContactList(predicate);
+
+        // Update the filtered lists accordingly.
+        // Flow of command returns back to the main dashboard.
+        Predicate<Contact> prevContactPred = model.getContactListPredicate();
+
+        if (prevContactPred != null) {
+
```
Comment (tjch-o) [inline src/main/java/seedu/address/logic/commands/contact/FindContactCommand.java:42]: random space should be removed

File: src/main/java/seedu/address/logic/commands/contact/FindContactCommand.java [hunk 2]
```diff
@@ -30,9 +33,23 @@ public FindContactCommand(NameContainsKeywordsPredicate predicate) {
...
+            model.updateFilteredContactList(predicate.and(prevContactPred));
+
+        } else {
+
```
Comment (tjch-o) [inline src/main/java/seedu/address/logic/commands/contact/FindContactCommand.java:46]: same as above

**Review by tjch-o — APPROVED** [review]
lgtm!

File: src/main/java/seedu/address/logic/commands/contact/FindContactCommand.java (hunk 1 above)
Comment (rionshocker) [reply src/main/java/seedu/address/logic/commands/contact/FindContactCommand.java]: Fixed! Thanks for spotting.

**Pull request merged**